DaytaCare/Models/Identity/ApplicationUser.cs

```diff
@@ -10,10 +10,10 @@ namespace DaytaCare.Models.Identity
 {
   public class ApplicationUser : IdentityUser
   {
-    [Required]
+    //[Required]
     public string FirstName {get; set;}
 
-    [Required]
+    //[Required]
```

Comment:
Line 13 and 16 we can do in the User DTO (I think Keith said this in the slack chat)

Reply:
Done!